_inc/client/plans/single-product-backup.jsx

```diff
@@ -155,6 +173,47 @@ function PlanRadioButton( {
 	);
 }
 
+function UpgradeButton( {
+	selectedUpgradeType,
+	upgradesAvailable,
+	billingTimeFrame,
+	currencyCode,
+	onClickHandler,
+} ) {
+	const selectedUpgrade = upgradesAvailable.find( ( { type } ) => type === selectedUpgradeType );
+	if ( ! selectedUpgrade ) {
+		return null;
+	}
+	const { link, name, fullPrice, discountedPrice } = selectedUpgrade;
+	let billingTimeFrameString = '';
+	if ( 'yearly' === billingTimeFrame ) {
+		billingTimeFrameString = 'year';
+	} else if ( 'monthly' === billingTimeFrame ) {
+		billingTimeFrameString = 'month';
+	}
+
+	const { symbol, integer, fraction } = getCurrencyObject(
+		discountedPrice || fullPrice,
+		currencyCode
+	);
+	const price = `${ symbol }${ integer }${ fraction }`;
```

Comment:
Any chance we could make sure this is localized? Not all languages use the currency code before the price, for example.

Reply:
I'd be super game, but I didn't see that anywhere and the `PlanPrice` component hardcoded it the same way.

Reply:
@jeherve - So I did some digging, and found that the [currency formatting library](https://github.com/Automattic/wp-calypso/blob/master/packages/format-currency/src/index.js) does not support currencies with the code after the price. I've opened [an issue](https://github.com/Automattic/wp-calypso/issues/38325) and can resolve when I get a moment- it also seems the NPM package is out of date there too.

Reply:
As an update: I added [a placeholder function](https://github.com/Automattic/jetpack/commit/61f85718cf3a715b1b85a13df7ab4883095d6c67#diff-7457a6cdabf24fa36923c20d4c9b101cR254) for this, and will work on fixing the currency formatting issue and retroactively patch this to remove my stub function. Just don't want it blocking things!